zavod/zavod/cli.py

```diff
@@ -119,6 +135,7 @@ def run(
     try:
         clear_store(dataset)
         view = get_view(dataset, external=False)
+        validate_dataset(dataset, view)
```

Comment:
I'm not sure this should be in here. It definitely wants to be in `run`, but maybe not in `export`?